lib/tasks/db.rake

```diff
@@ -7,7 +7,7 @@ namespace :db do
     Rake::Task['invoker:start_all'].invoke
   end
 
-  task seed: ['rabbitmq:purge', 'chewy:reset'] do
+  task :seed do
```

Comment:
If you remove the prerequisites of the task, you can remove the whole task. Then, it is simply inherited from the models.